.env.example

```
@@ -1,14 +1,4 @@
-# Database Configuration
-# For production, use Turso
-TURSO_DATABASE_URL="libsql://db-name-user.turso.io"
-TURSO_AUTH_TOKEN=""
-
-# For local development, use SQLite file
-# This will be used in development mode when present
-LOCAL_DATABASE_URL="file:local.db"
-
-# Resend API key for email sending
+DATABASE_URL="libsql://db-name-user.turso.io"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)